VMBackup/main/ExtensionErrorCodeHelper.py

```diff
@@ -42,7 +42,8 @@ class ExtensionErrorCodeEnum():
     FailedPrepostPluginConfigPermissionError = 317
     FailedPrepostPluginConfigOwnershipError = 318
     FailedGuestAgentInvokedCommandTooLate = 402
-    
+    FailedInputMismatch = 405
```

Comment:
Are these error codes supposed to corelate to http status codes? If they do a simple 400 should work.

Reply:
As we are failing the extension specifically because there is a mismatch between the expected and received parameters, I think this error code would be able to point that.

Reply:
Input mismatch can happen due to multiple conditions. The name should specifically mention the condition that is failing.

Reply:
Hey @arisettisanjana , I still think using 'mismatch' in the error code can be a misnomer. A good example of mismatch would be-> to have one taskId in input but different taskId in output for the same run. This would be a mismatch. 
This DataDiskLunList field being empty when atleast one disk is excluded is a case of poor input creation by the requestor. 
Some suggestions: 'FailedInvalidDataDiskLunList' or 'FailedInconsistentDataDiskLunList'

Reply:
sure